### ISSUE TITLE
Focal loss for imbalanced classs

### ISSUE THREAD

**Issue by programmeriop** [ultralytics/ultralytics#2276]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How do can I  tune the YOLOv8 (detect) parameters to use focal loss?  
I also checked that the focal loss function is defined in yolov8, but I don't see anything in the documentation on how to use class balance loss, if you have a reference or method, could you please let me know?

**Comment by Laughing-q** [comment]
@programmeriop yes the function is in the code, but it doesn't show improvement in our experiments so we remove the related arg. You might need to modify the loss function in ultralytics/yolo/v8/detect/train.py if you want to use it. :)
